.travis.yml

```diff
@@ -4,6 +4,8 @@ matrix:
   include:
     - python: 2.7
       env: TOXENV=py27
+    - python: 3.5
+      env: TOXENV=py35
```

Comment:
Do we need to still support 3.5? I would prefer to keep our support surface area as small as possible. I don't think we need to add this back.

Reply:
You mentioned earlier that this might be needed for Winston support.

If that is no longer necessary, then we can trim this back.